konf/site.yaml

```diff
@@ -11,6 +11,16 @@ env:
       key: google-custom-search-key
       name: google-api
 
+  - name: DISCOURSE_API_KEY
+    secretKeyRef:
+      key: maas-api-key
+      name: discourse-api
+
+  - name: DISCOURSE_API_USERNAME
+    secretKeyRef:
+      key: maas-api-username
+      name: discourse-api
+
```

Comment:
can you try with this https://github.com/canonical/ubuntu.com/blob/main/konf/site.yaml#L20?